docs/data/material/components/buttons/InputFileUpload.js

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import SvgIcon from '@mui/material/SvgIcon';
+
+const VisuallyHiddenInput = styled('input')`
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  white-space: nowrap;
+  width: 1px;
+`;
+
+export default function InputFileUpload() {
+  return (
+    <Button
+      component="label"
+      variant="contained"
+      startIcon={
+        <SvgIcon>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            strokeWidth={1.5}
+            stroke="currentColor"
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              d="M12 16.5V9.75m0 0l3 3m-3-3l-3 3M6.75 19.5a4.5 4.5 0 01-1.41-8.775 5.25 5.25 0 0110.233-2.33 3 3 0 013.758 3.848A3.752 3.752 0 0118 19.5H6.75z"
+            />
+          </svg>
+        </SvgIcon>
+      }
```

Comment:
```suggestion
      startIcon={
        <CloudUploadIcon/>
      }
```

It's simpler to pull off the icon from the icons package! Make sure to update the imports, too!